backend/src/main/java/com/google/sps/data/TimeHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+
+import java.time.ZonedDateTime;
+
+/**
+ * Class used to set reference date/time for testing and convert input timeAvailableUntil to date
+ * timestamp
+ */
+public final class TimeHelper {
+
+  private static ZonedDateTime dateTime;
+
+  /** Constructor */
+  public TimeHelper(ZonedDateTime dateTime) {
+    this.dateTime = dateTime;
+  }
+
+  /** Return today's date with time of hour:minute */
+  public static ZonedDateTime getNewTimeToday(int hour, int minute) {
```

Comment:
Having an instance constructor set a static field is error prone

```
TimeHelper th1 = TimeHelper(/* December 4, 2019 */)
TimeHelper th2 = TimeHelper(/* June 27, 2053 */)
th1.getNewTimeToday(12, 25)
```

What does that evaluate to? I would expect 2019-12-04 12:25 but it would actually be 2053-06-27 12:25.

I recommend making both `dateTime` and `getNewTimeToday` non-static.